shared/constants/chat2/message.js

```diff
@@ -356,7 +356,7 @@ const makeMessageSystemChangeRetention: I.RecordFactory<MessageTypes._MessageSys
     isInherit: false,
     isTeam: false,
     membersType: 0,
-    policy: RPCChatTypes.RetentionPolicy,
+    policy: RPCChatTypes.commonRetentionPolicyType.none,
```

Comment:
This is typed as
```
export type RetentionPolicy = {typ: 1, retain: ?RpRetain} | {typ: 2, expire: ?RpExpire} | {typ: 3, inherit: ?RpInherit} | {typ: 4, ephemeral: ?RpEphemeral}
```
it's probably better to make this `{typ: RPCChatTypes.commonRetentionPolicyType.none}`. I think this should be changed to the UI type but that doesn't need to be done here.

Reply:
`canChange` in this object actually isn't in `_MessageSystemChangeRetention` either. TIL flow doesn't check these.